src/features/user/ChatwootWidget/index.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+class ChatwootWidget extends React.Component {
+  componentDidMount() {
+    // Add Chatwoot Settings
+    window.chatwootSettings = {
+      hideMessageBubble: false,
+      position: 'right', // This can be left or right
+      locale: 'en', // Language to be set
```

Comment:
--> `locale: user.locale ? user.locale : 'en',`

Reply:
I localized the widget with `window.$chatwoot.setLocale(user.locale ? user.locale : 'en');`, but the remaining custom welcome message at the top of the chat window is still in English:

<img width="1625" alt="Bildschirmfoto 2022-01-04 um 15 31 55" src="https://user-images.githubusercontent.com/1532418/148074583-e2ce3169-6b90-4caa-8751-4238c9191f09.png">

Reply:
I think that's something we'll have to live with unless we use a different `websiteToken` for German.

Reply:
using language stored for UI in localStorage